//
//  SharedConstants.h
//  JSONExporter
//
//  Created by lizhao on 16/3/26.
//  Copyright © 2016年 lizhao. All rights reserved.
//

#ifndef SharedConstants_h
#define SharedConstants_h

const NSString *elementType = @"<!ElementType!>";
const NSString *modelName = @"<!ModelName!>";
const NSString *modelWithParentClassName = @"<!ParentClass!>";
const NSString *varName = @"<!VarName!>";
const NSString *capitalizedName = @"<!CapitalizedVarName!>";
const NSString *varType = @"<!VarType!>";
const NSString *varTypeReplacement = @"<!VarBasicTypeReplacement!>";
const NSString *capitalizedVarType = @"<!CapitalizedVarType!>";
const NSString *lowerCaseVarType = @"<!LowerCaseVarType!>";
const NSString *lowerCaseModelName = @"<!LowerCaseModelName!>";
const NSString *jsonKeyName = @"<!JsonKeyName!>";
const NSString *constKeyName = @"<!ConstKeyName!>";
const NSString *additionalCustomTypeProperty = @"<!AdditionalForCustomTypeProperty!>";

#endif /* SharedConstants_h */
